src/pages/organize/[orgId]/projects/calendar/index.tsx

```diff
@@ -25,17 +25,14 @@ const scaffoldOptions = {
 export const getServerSideProps: GetServerSideProps = scaffold(async (ctx) => {
   const { orgId } = ctx.params!;
 
-  await ctx.queryClient.prefetchQuery(
-    ['org', orgId],
-    getOrg(orgId as string, ctx.apiFetch)
+  const apiClient = new BackendApiClient(ctx.req.headers);
+  const organization = await apiClient.get<ZetkinOrganization>(
+    `/api/orgs/${orgId}`
   );
-  const orgState = ctx.queryClient.getQueryState(['org', orgId]);
 
-  if (orgState?.status === 'success') {
+  if (organization) {
```

Comment:
This will always be true. If we got this far, it's because the API call succeeded. If it failed, it will have thrown an error and that error is not handled anywhere, resulting in a 500 (if it gets this far).